js/foundation.drilldown.js

```diff
@@ -85,7 +85,11 @@ class Drilldown {
       var $menu = $(this),
           $back = $menu.find('.js-drilldown-back');
       if(!$back.length){
-        $menu.prepend(_this.options.backButton);
+        if(_this.options.backButtonPosition == 'bottom'){
+          $menu.append(_this.options.backButton);
+        } else {
+          $menu.prepend(_this.options.backButton);
+        }
```

Comment:
Right, totally forgot about that, it's been a couple days haha, hows this? 

````
switch (this.buton.backButtonPosition) {
    case "bottom":
        $menu.append(_this.options.backButton);
        break;
     case "top";
        $menu.prepend(_this.options.backButton);
        break;
     default:
        console.log("Unsupported backButtonPosition value '" + backButtonPosition + "'");
}
````